.github/actions/BuildJlm/action.yml

```diff
@@ -19,7 +19,7 @@ inputs:
 
   cxx:
     description: "Set C++ compiler. Default is 'clang++'."
-    default: "clang++-16"
+    default: "clang++-17"
```

Comment:
I think it is possible to just say `clang++` here, avoiding the version change on the next LLVM update.

Reply:
Changed to clang++